catalog/transformer/catalog-transformer-streaming-lib/src/main/java/org/codice/ddf/transformer/xml/streaming/lib/XmlInputTransformer.java

```diff
@@ -259,7 +248,7 @@ public MetacardType getMetacardType() {
      * @return a DynamicMetacardType that describes the type of metacard that is created in this transformer
      */
 
-    public DynamicMetacardType defineMetacardType() {
+    public MetacardType getMetacardType() {
         Set<AttributeDescriptor> attributeDescriptors = new HashSet<>();
```

Comment:
I know you didn't change this but if we know the size before hand lets initialize the hashset since the default is 18 and your only using 2.